<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PostDialog</class>
 <widget class="QDialog" name="PostDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>596</width>
    <height>550</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="likeCnt">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>230</y>
     <width>31</width>
     <height>32</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QPushButton" name="laughCnt">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>230</y>
     <width>31</width>
     <height>32</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QPushButton" name="dlBtn">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>230</y>
     <width>31</width>
     <height>32</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="text">
    <string>üëé</string>
   </property>
  </widget>
  <widget class="QPushButton" name="likeBtn">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>230</y>
     <width>31</width>
     <height>32</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="text">
    <string>üëç</string>
   </property>
  </widget>
  <widget class="QPushButton" name="dlCnt">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>230</y>
     <width>31</width>
     <height>32</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QPushButton" name="surBtn">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>230</y>
     <width>31</width>
     <height>32</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="text">
    <string>üò≤</string>
   </property>
  </widget>
  <widget class="QPushButton" name="laughBtn">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>230</y>
     <width>31</width>
     <height>32</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="text">
    <string>üòÇ</string>
   </property>
  </widget>
  <widget class="QLabel" name="FdLbl">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>70</y>
     <width>321</width>
     <height>131</height>
    </rect>
   </property>
   <property name="text">
    <string>Full Post Display</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="surCnt">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>230</y>
     <width>31</width>
     <height>32</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="reactTbl">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>310</y>
     <width>256</width>
     <height>192</height>
    </rect>
   </property>
   <property name="editTriggers">
    <set>QAbstractItemView::NoEditTriggers</set>
   </property>
   <column>
    <property name="text">
     <string>Users Reacted:</string>
    </property>
   </column>
  </widget>
  <zorder>FdLbl</zorder>
  <zorder>likeCnt</zorder>
  <zorder>laughCnt</zorder>
  <zorder>dlBtn</zorder>
  <zorder>likeBtn</zorder>
  <zorder>dlCnt</zorder>
  <zorder>surBtn</zorder>
  <zorder>laughBtn</zorder>
  <zorder>surCnt</zorder>
  <zorder>reactTbl</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
